packages/ui/src/components/va-alert/VaAlert.stories.ts

```diff
@@ -1,12 +1,217 @@
-import { defineComponent } from 'vue'
-import VaAlert from './VaAlert.demo.vue'
+import { VaAlert } from './'
+import { VaButton } from '../va-button'
+import { userEvent } from '../../../.storybook/interaction-utils/userEvent'
+import { within } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
 
 export default {
   title: 'VaAlert',
   component: VaAlert,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
+export const Default = {
+  args: {
+    description: 'Message',
+  },
+}
+
+export const Dense = {
+  args: {
+    dense: true,
+    description: 'Message',
+  },
+}
+
+export const Title = {
+  args: {
+    title: 'Title',
+    description: 'Message',
+  },
+}
+
+export const Center = {
+  args: {
+    center: true,
+    title: 'Title',
+    description: 'Message',
+  },
+}
+
+export const Icon = {
+  args: {
+    icon: 'Icon',
+    description: 'Message',
+  },
+}
+
+export const Outline = {
+  args: {
+    outline: true,
+    description: 'Message',
+  },
+}
+
+export const Closeable = () => ({
+  components: { VaAlert },
+  template: `
+    [true]
+    <va-alert closeable>
+      Message
+    </va-alert>
+    [false]
+    <va-alert>
+      Message
+    </va-alert>
+  `,
+})
+
+export const CloseText = () => ({
```

Comment:
We need another story for `closeIcon`. This prop is not mentioned in docs, so we need to mention it there as well - https://ui.vuestic.dev/ui-elements/alert#props.